sdk/metrics/build.gradle.kts

```diff
@@ -23,4 +23,6 @@ dependencies {
   testImplementation(project(":sdk:metrics-testing"))
   testImplementation(project(":sdk:testing"))
   testImplementation("com.google.guava:guava")
+
+  jmh(project(":sdk:trace"))
```

Comment:
I think it'd be better to put the benchmarks in `sdk:all` if possible this looks like an unexpected coupling of the SDK projects

Reply:
JMH for trace also uses metrics sdk right now: https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/trace/build.gradle.kts#L37

This is in-line with our status-quo, but if you want to change that, I'd suggest outside this PR and we do it consistently.